scripts/RetentionScript/README.md

```diff
@@ -0,0 +1,68 @@
+
+# About the scripts:
+These scripts are used to cleanup the storage configured for BRS service in service fabric. It allows you to delete data older than specific date time.
+
+## Usage
+This script could be used in case of both type of supported storages:
+1) Azure blob store
+2) File Share
+ How to use with Azure Storage:
+ You can run the script by providing single azure storage connection string as parameter or you can also provide azure account name and key separately.
+ Examples:
+1) The below example will delete all the backups in the storage(specifed with connection string) of the service with serviceid "WebReferenceApplication~RestockRequestManager".
+```powershell
+ .\RetentionScript.ps1 -DateTimeBefore "2018-06-18 23.44.03Z" -ConnectionString "your-connection-string" -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -SSLCertificateThumbPrint "Client#Certificate#Thumbpring" -ServiceId "WebReferenceApplication~RestockRequestManager"
```

Comment:
WebReferenceApplication~RestockRequestManager -> This is not the standard way of specifying service ID in Service Fabric. You should explain here that if your service name is fabric:/WebReferenceApplication/RestockRequestManager give the input as WebReferenceApplication~RestockRequestManager